purge-tool/src/main/java/dk/dbc/holdingsitems/purge/PurgeMain.java

```diff
@@ -105,7 +91,7 @@ public static void main(String[] args) throws OpenAgencyException, IOException {
 
                 jpa.run(em -> {
                     HoldingsItemsDAO dao = HoldingsItemsDAO.newInstance(em, trackingId);
-                    Purge purge = new Purge(em, dao, queue, agencyName, agencyId, dryRun);
+                    Purge purge = new Purge(em, dao, queue, agencyName, agencyId, removeFirstAcquisitionDate, dryRun);
```

Comment:
Så hvad? removeFirstAD virkede bare ikke før?

Reply:
Åbentbart... der var en metode til at gøre det, den blev bare ikke kaldt uden for `*IT.java`